frontend/javascripts/admin/user/user_list_view.js

```diff
@@ -303,42 +285,6 @@ class UserListView extends React.PureComponent<PropsWithRouter, State> {
           >
             Change Experience
           </Button>
-          {this.props.activeUser.isAdmin ? (
-            <React.Fragment>
-              <Button
-                onClick={() =>
-                  Modal.confirm({
-                    title: messages["users.grant_admin_rights_title"],
```

Comment:
There was an extra confirmation (modal) needed to grant or revoke admin rights, before, but I couldn't find this functionality in the new version. Did you discuss and agree to remove this?

Reply:
No we did not discuss anything I just removed it without thinking :see_no_evil:. Do you think we should keep a confirmation, if the permissions (not team membership) of a user are changed?